pkg/multus/multus.go

```diff
@@ -503,6 +504,22 @@ func cmdPluginErr(k8sArgs *types.K8sArgs, confName string, format string, args .
 	return logging.Errorf(msg+format, args...)
 }
 
+func isCriticalRequestRetriable(err error) bool {
+	logging.Debugf("isCriticalRequestRetriable: %v", err)
+	errorTypesAllowingRetry := []func(error) bool{
+		errors.IsServiceUnavailable, errors.IsInternalError, k8snet.IsConnectionReset, k8snet.IsConnectionRefused}
+	for _, f := range errorTypesAllowingRetry {
+		if f(err) {
+			return true
+		}
+	}
+	return false
+}
+
+func generateRetryBackoff(interval time.Duration, timeout time.Duration) wait.Backoff {
```

Comment:
Currently we use same duration (250msec) and timeout (5sec) for all function. Can we remove both from its arugment if possible? (or make everything in argument?)

Reply:
@maiqueb Thank you for your reply. I understand that, yeah, it is pretty trade-off. Currently we do not have strict timeout tuning and we do not have strict requirement from other components, hence I guess we don't need to make it parameter for now.

Once we face such case, then we just change it, I guess.